.scripts/inject_version_warning.py

```
@@ -0,0 +1,379 @@
+                config = yaml.safe_load(f)
+            else:
+                import json
+
+                config = json.load(f)
+
+            info["version"] = str(config.get("version", "unknown"))
+            info["name"] = config.get("name", info["name"])
+            info["slug"] = config.get("slug", info["slug"])
+    except (yaml.YAMLError, OSError, ValueError) as e:
+        print(f"⚠️ Could not read config: {e}")
+    except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)